src/index.tsx

```diff
@@ -1,11 +1,10 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import {RBSearchWidget} from "./components/rb_search_widget";
 
 document.addEventListener('DOMContentLoaded', () => {
     ReactDOM.render(
-        <div>
-            Hello World
-        </div>,
+        <div><RBSearchWidget {...{...{state: 'Texas', city: 'Dallas', zipcode: '11100'}, newTab: false}}/></div>,
```

Comment:
You can just do `{...{state: 'Texas', city: 'Dallas', zipcode: '11100', newTab: false}}`